reventlov/plugins/trello/__init__.py

```diff
@@ -80,46 +80,67 @@ def get_board(self, board_name):
                 break
         return board_found
 
+    def list_orgs(self):
+        msg = '\n'.join([
+            f'- *{org_name}*' if org_name == self.organization.name
+            else f'- {org_name}'
+            for org_name in self.org_names
+        ])
+        return msg, ParseMode.MARKDOWN
+
+    def list_boards(self):
+        msg = '\n'.join([
+            f'- {board_name}'
+            for board_name in self.board_names
+        ])
+        return msg, ParseMode.MARKDOWN
+
+    def list_column_cards(self, column):
+        msg = '\n'.join([
+            f'    + {card.name}'
+            for card in column.list_cards()
+        ])
+        return msg
+
+    def list_board_columns(self, board_name):
+        msg = f'No such board `{board_name}`'
+        board = self.get_board(board_name)
+        if board is not None:
+            msg = '\n'.join([
+                f'- {column.name} ({column.cardsCnt()} cards) ' \
+                f'\n{self.list_column_cards(column)}'.strip()
+                for column in board.open_lists()
+            ])
+        return msg, ParseMode.HTML
```

Comment:
It should be `MARKDOWN` instead of `HTML`, but it is failing because of some issue with entities and encoding I don't have very clear.

Reply:
I created issue #26 to address this.